editor/editor_export.h

```diff
@@ -483,4 +485,46 @@ class EditorExportTextSceneToBinaryPlugin : public EditorExportPlugin {
 	EditorExportTextSceneToBinaryPlugin();
 };
 
+class EditorExportConsolePlaceholder : public EditorExportPlatform {
+	GDCLASS(EditorExportConsolePlaceholder, EditorExportPlatform);
+
+public:
+	String console_name;
+	String console_owner;
+	String console_os_name;
+	String console_licensing;
+	Ref<Texture2D> logo;
+
+	virtual String get_name() const override {
+		return console_name;
+	}
+	virtual String get_os_name() const override {
+		return console_os_name;
+	}
+	virtual Ref<Texture2D> get_logo() const override {
+		return logo;
+	}
+
+	bool can_export(const Ref<EditorExportPreset> &p_preset, String &r_error, bool &r_missing_templates, bool &r_missing_console) const override {
+		r_error = vformat(TTR("Exporting to %s requires getting licensed with %s, then obtaining export templates from a third-party company (or porting it yourself)."), console_name, console_licensing);
```

Comment:
I was thinking about:

As %s does not authorize public distribution of their SDKs, you need to get licensed with them first. Afterwards, export templates are commercially available from third party companies. More information here...

Reply:
I can't check how the message looks in practice, so I need to check whether it'll wrap automatically or not. If not, I'll need to use `String::word_wrap`.

Reply:
It does not word wrap automatically but respect `\n` in the text, current line is a bit too long as well.

<img width="1065" alt="Screenshot 2020-09-27 at 19 18 16" src="https://user-images.githubusercontent.com/7645683/94370107-8a77e480-00f6-11eb-8a71-8efb32f14973.png">

```suggestion
		r_error = vformat(TTR("As %s does not authorize public distribution of their SDKs, you need to get licensed with them first.\nAfterwards, export templates are commercially available from third party companies."), console_licensing);
```

<img width="767" alt="Screenshot 2020-09-27 at 19 22 19" src="https://user-images.githubusercontent.com/7645683/94370142-d62a8e00-00f6-11eb-9ee4-a03b6d5a7c9d.png">